estuary_updater/handlers/errata.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+from estuary_updater.handlers.base import BaseHandler
+from estuary.models.errata import Advisory
+from estuary.models.bugzilla import BugzillaBug
+
+import requests
+import requests_kerberos
+import iso8601
+
+
+class ErrataHandler(BaseHandler):
+    """A handler for dist-git related messages."""
+
+    @staticmethod
+    def can_handle(msg):
+        """
+        Determine if this handler can handle this message.
+
+        :param dict msg: a message to be analyzed
+        :return: a bool based on if the handler can handle this kind of message
+        :rtype: bool
+        """
+        return msg['topic'] == '/topic/VirtualTopic.eng.errata.activity.status'
+
+    def handle(self, msg):
+        """
+        Handle an Errata tool message and update Neo4j if necessary.
+
+        :param dict msg: a message to be processed
+        """
+        advisory_id = msg['body']['headers']['errata_id']
+
+        url = 'https://errata.devel.redhat.com/api/v1/erratum/{}'.format(advisory_id)
+        response = requests.get(url, auth=requests_kerberos.HTTPKerberosAuth())
+        advisory_json = response.json()
+
+        # The way DateTime objects are handled will need to be changed once we stop using the API,
+        # as the strings are formatted differently in the message bus messages
+        actual_ship_date = list(advisory_json['errata'].values())[0]['actual_ship_date']
```

Comment:
Instead of `list(advisory_json['errata'].values())[0]` you can get the advisory type from the UMB message and then make that lower case and use that as the key.